saw-core/src/SAWCore/Simulator/Value.hs

```diff
@@ -5,6 +5,7 @@
 {-# LANGUAGE TupleSections #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE UndecidableInstances #-}  -- For `Show` instance, it's OK.
+{-# OPTIONS_GHC -Wno-unused-top-binds #-}
```

Comment:
What is the motivation behind disabling this warning?

Reply:
Oops, I added that temporarily while I experimented with trimming the export list. I'll fix that.